README.md

```diff
@@ -227,10 +227,21 @@ $ npm run component-gen
 
 ## ➤ How to release
 
-As mentioned before, we are using [lerna](https://github.com/lerna/lerna), to be able to publish you need enable your [two-factor authentication in npm](https://docs.npmjs.com/configuring-two-factor-authentication)
+We are using [lerna to publish](https://github.com/lerna/lerna/tree/master/commands/publish#readme) our elements
+
+- Create a branch for releasing。 `git branch release`
+- `git push release` and set the upstream.
+- `lerna publish`
+- If you have problems while releasing. The following may help
 
 ```shell
-$ NPM_CONFIG_OTP=123456 lerna publish`   # Push the packages to registry.npmjs.org
+# delete local tag '12345'
+git tag -d 12345
+# delete remote tag '12345' (eg, GitHub version too)
+git push origin :refs/tags/12345
+# alternative approach
+git push --delete origin tagName
+git tag -d tagName
```

Comment:
One of the way to deleting the tag would be enough I think! Or maybe even moving this information to wiki page?
I think keeping the readme file as clean and lean as possible give it more chance to actually be read by the users not just scrolling through it

Reply:
👍